module1/homework/make_unique/make_unique-ut.cpp

```diff
@@ -0,0 +1,64 @@
+#include <gtest/gtest.h>
+#include <memory>
+
+#include "make_unique.hpp"
+
+constexpr int testValue{2020};
+constexpr int intValue{1};
+constexpr double doubleValue{1.5};
+constexpr bool boolValue{true};
+
+class MyTestType {
+public:
+    MyTestType() = default;
+    MyTestType(int firstValue, double secondValue, bool thirdValue)
+        : intValue_(firstValue), doubleValue_(secondValue), boolValue_(thirdValue) {}
+
+    bool operator==(const MyTestType& other) const {
+        return intValue_ == other.intValue_ &&
+               doubleValue_ == other.doubleValue_ &&
+               boolValue_ == other.boolValue_;
+    }
+
+private:
+    int intValue_{intValue};
+    double doubleValue_{doubleValue};
+    bool boolValue_{boolValue};
+};
+
+TEST(make_unique, shouldMakeUniqueToInt) {
+    std::unique_ptr<int> uniqZero{new int{}};
+    auto testUniquePtr = cs::make_unique<int>();
+    ASSERT_EQ(*uniqZero, *testUniquePtr);
+}
+
+TEST(make_unique, shouldMakeUniqueToInt2020) {
+    std::unique_ptr<int> uniqEleven{new int{testValue}};
+    auto testUniquePtr = cs::make_unique<int>(testValue);
+
+    ASSERT_EQ(*testUniquePtr, testValue);
+    ASSERT_EQ(*uniqEleven, *testUniquePtr);
+}
+
+TEST(make_unique, shouldMakeUniqueToEmpytyMyTestTypeObject) {
+    std::unique_ptr<MyTestType> uniqMyTestType{new MyTestType{}};
+    auto testUniquePtr = cs::make_unique<MyTestType>();
+    ASSERT_EQ(*uniqMyTestType, *testUniquePtr);
+}
+
+TEST(make_unique, shouldMakeUniqueToMyTestTypeObject) {
+    std::unique_ptr<MyTestType> uniqMyTestType{new MyTestType{2, 2.5, 0}};
+    auto testUniquePtr = cs::make_unique<MyTestType>(2, 2.5, 0);
```

Comment:
The last argument is bool
```suggestion
    auto testUniquePtr = cs::make_unique<MyTestType>(2, 2.5, false);
```